src/components/DownscaledLayers.vue

```diff
@@ -0,0 +1,210 @@
+               <div>
+                 <v-btn @click="save">
+                   <v-icon class="save-icon">
+                     save_alt
+                   </v-icon>
+                 </v-btn>
+               </div>
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </v-flex>  
+    </v-layout>  
+  </div>
+</template>
+<script>
+import buildWmsLayer from  "@/libs/build-wms-layer";
+import layers from "@/config/datalayers.json";
+import { getCoverage } from "@/libs/get-coverage";
+import { mapState } from 'vuex';
+export default {
+  props: {
+    visibility: {
+      type: Boolean
+      },
+  },
+  data() {
+    return {
+      layers,
+    };
+  },
+  computed: {
+    ...mapState({
+      selectedLayer: state => state.selectedLayer
+    }),
+  },
+  methods: {
+    setwmslayer(layer) {
+      if (layer.active) {
+        this.$store.commit("SET_SELECTED_LAYER", layer)
+        this.$store.commit("REMOVE_COMPARE_LAYER", layer)
+        this.$store.commit("ADD_COMPARE_LAYER", layer)
+        if (this.visibility===true){
+          layer = this.updateVisibility(layer)
+        }
+        const wmsLayer = buildWmsLayer(layer)
+        this.$store.commit("REMOVE_WMS_LAYER", layer.id)
+        this.$store.commit("ADD_WMS_LAYER", wmsLayer)
+ 
+        layer.original = {
+            ...layer.original,
+            time:layer.time
+          }
+        const wmsLayerOriginal = buildWmsLayer(layer.original)
+        
+        this.$store.commit("REMOVE_WMS_LAYER", layer.original.id)
+        this.$store.commit("ADD_WMS_LAYER", wmsLayerOriginal)
+
+      } 
+      else {
+        this.$store.commit("REMOVE_WMS_LAYER", layer.id) 
+        this.$store.commit("REMOVE_COMPARE_LAYER", layer.id)
+        this.$store.commit("REMOVE_WMS_LAYER", layer.original.id)
+      }
+    },
+  
+    updateVisibility(layer){  
+          return { 
+            ...layer,
+            visibility:'none',
+            original: {
+              ...layer.original,
+              ["visibility"]: "visible" 
+            }
+          } 
+    },
+    save() { 
+      let url = this.selectedLayer.url
+      let layer = this.selectedLayer.layer
+      let bbox = this.selectedLayer.bbox
+      let time = this.selectedLayer.time
+      let downloadLink = getCoverage({url, layer, bbox, time})
+      window.open(
+        downloadLink,
+        '_blank'
+      ) 
+    }
+}
+}
+</script>
+<style>
+.title {
+  color: white;
+}
+#dragicon {
+  margin: auto;
+}
+
+#draggable {
+  width: 100%;
+}
+
+.v-expansion-panel {
+  box-shadow: none;
+  webkit-box-shadow: none;
+}
+.fa-grip-vertical:hover {
+  color: black;
+  cursor: grab;
+}
+
+.infodiv {
+  .wordwrap {
+    white-space: pre-wrap; /* CSS3 */
+    white-space: -moz-pre-wrap; /* Firefox */
+    white-space: -pre-wrap; /* Opera <7 */
+    white-space: -o-pre-wrap; /* Opera 7 */
+    word-wrap: break-word; /* IE */
+  }
+}
+/* Customize the switch buttons */
```

Comment:
why? this falls outside the use of a design framework i'd say ( the whole reason we are not making it from vanilla js)

use vuetify classes for margin/padding: https://vuetifyjs.com/en/styles/spacing/ (in this case ma-auto)